examples/fileuploader.hs

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env stack
```

Comment:
We need to add copyright header to all source code files. This will be addressed in a separate PR fixing #16